app/models/service_provider_request.rb

```diff
@@ -20,6 +20,7 @@ def initialize(
     # rubocop:disable Lint/UnusedMethodArgument
     ial: nil,
     aal: nil,
+    enhanced_ipp_required: false,
```

Comment:
2 things here:

1. This is going to be problematic in the 50/50 state because this could get serialized to the session by a new instance and read from an old instance. That would result in an `ArgumentError` because the old instance will not know about this arg.